cli/command/engine/activate_unix.go

```diff
@@ -0,0 +1,13 @@
+// +build !windows
+
+package engine
+
+import (
+	"golang.org/x/sys/unix"
+)
+
+var (
+	isRoot = func() bool {
```

Comment:
I don't think putting a function as global variable to swap it in tests is a clean way to mock things. It does not scale, it's not future proof as any piece of code can erase it without notice. You should use an interface instead, or refactor your code so it is testable without this kind of "trick".